src/cpp/rtps/network/NetworkFactory.cpp

```diff
@@ -34,10 +34,36 @@ namespace rtps {
 
 using SendResourceList = fastdds::rtps::SendResourceList;
 
-NetworkFactory::NetworkFactory()
+NetworkFactory::NetworkFactory(
+        const RTPSParticipantAttributes& PParam)
     : maxMessageSizeBetweenTransports_(std::numeric_limits<uint32_t>::max())
     , minSendBufferSize_(std::numeric_limits<uint32_t>::max())
 {
+    const std::string* allow_metatraffic = nullptr;
```

Comment:
Should we rename this local var as well?
```suggestion
    const std::string* enforce_metatraffic = nullptr;
```

Reply:
f6bef17542544dac44f25b97f942fe09f6642c1a